src/index.html

```diff
@@ -11,7 +11,161 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      name="layoutHtmlForm"
+      method="post"
+    >
+      <fieldset class="formFieldsets">
+        <legend>Personal Information</legend>
+        <label for="surname" class="labels">
```

Comment:
Attribute "for" works in link with id, not name. Please add correct id to your inputs so that labels work properly